nifs/atomvm_ssd1306.c

```diff
@@ -215,6 +232,43 @@ static esp_err_t write_page_command(i2c_port_t i2c_num, uint8_t *page, uint8_t p
     return err;
 }
 
+static term nif_ssd1306_set_inversion(Context *ctx, int argc, term argv[])
+{
+    TRACE(TAG ": nif_ssd1306_set_inversion\n");
+    UNUSED(argc);
+
+    term i2c_num_term = argv[0];
+    VALIDATE_VALUE(i2c_num_term, term_is_atom);
+    term mode_term = argv[1];
+    VALIDATE_VALUE(i2c_num_term, term_is_atom);
+
+    i2c_port_t i2c_num = get_i2c_port_num(ctx->global, i2c_num_term);
+    if (i2c_num == I2C_NUM_MAX) {
+        ESP_LOGE(TAG, "Invalid I2C portnum");
+        RAISE_ERROR(BADARG_ATOM);
+    }
+
+    bool inversion_mode;
+    switch(mode_term){
+      case TRUE_ATOM:
```

Comment:
This might compile, and might event test ok, but case should only be used with integers, which `TRUE_ATOM` is not. I am sure this is causing a compiler warning. I just realized this repo does not have any build tests or other CI checks.

What you can do is use:
```
switch(term_to_atom_index(mode_term)) {
    case TRUE_ATOM_INDEX:
    ...
    case FALSE_ATOM_INDEX:
```
This will use [term_to_atom_index](https://www.atomvm.net/doc/main/apidocs/libatomvm/src/term.h.html#c.libAtomVM.term.h.term_to_atom_index) convert the atom strings to their index id, which are defined in this case in AtomVM/src/libAtomVM/defaultatoms.h as:
```
#define FALSE_ATOM_INDEX 0
#define TRUE_ATOM_INDEX 1
```

Reply:
I really appreciate this example, it was not so easy for me to figure it out myself.
I've done exactly as you proposed.